backend/de.metas.business/src/main/java/de/metas/order/compensationGroup/OrderGroupRepository.java

```diff
@@ -770,15 +766,20 @@ public GroupTemplateId getGroupTemplateId(@NonNull final GroupId groupId)
 	public I_C_OrderLine createRegularLineFromTemplate(
 			@NonNull final GroupTemplateRegularLine from,
 			@NonNull final I_C_Order targetOrder,
-			@Nullable final ConditionsId contractConditionsId)
+			@Nullable final ConditionsId contractConditionsId,
+			final @NonNull RetrieveOrCreateGroupRequest request)
```

Comment:
contractConditionsId is already part of the request.
either u are passing all params, either u are passing the request

Reply:
removed contractConditionsId